http-clients/netty-nio-client/src/test/java/software/amazon/awssdk/http/nio/netty/internal/http2/utils/Http2TestUtils.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: believe we remove all the copyright years from the license headers now (#1655)